test/integration-tests/IntegrationTests.Helpers/ProcessHelper.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.Diagnostics;
+using System.Text;
+using System.Threading;
+
+namespace IntegrationTests.Helpers
+{
+    /// <summary>
+    /// Drains the standard and error output of a process
+    /// </summary>
+    public class ProcessHelper : IDisposable
+    {
+        private const int DefaultTimeoutMinutes = 30;
```

Comment:
Do we really want to have that long timeout for draining the output?
Maybe 1 minute?

Reply:
Output is generated through the lifetime of the sample. I guess this and process timeout is pretty much connected.

Reply:
Gotcha then maybe 5 min as well?

Reply:
ok, I changed both to 5 mins, let's try, we can adjust always later if there are issues.